src/server/cluster/cluster_family.cc

```diff
@@ -396,8 +396,6 @@ void ClusterFamily::DflyCluster(CmdArgList args, ConnectionContext* cntx) {
     return DflyClusterStartSlotMigration(args, cntx);
```

Comment:
also start slot migration need to be removed dont forget

Reply:
I will have another PR for migration initiation refactoring.